Makefile

```
@@ -1,30 +1,39 @@
 PATH := node_modules/.bin/:$(PATH)
 
-start:
-	node server.js
-start-nodemon:
-	nodemon -L server.js
+start: start-support
+	@node server.js
```

Alternative to the other change:
```suggestion
ifdef NODEMON
start: start-nodemon
else
start: start-support
	@node server.js
endif
```

---

I went with the other one.